Library/Homebrew/service.rb

```diff
@@ -124,35 +124,49 @@ def restart_delay(value = nil)
       end
     end
 
-    sig { params(type: T.nilable(String)).returns(T.nilable(String)) }
-    def process_type(type = nil)
-      case T.unsafe(type)
+    sig { params(value: T.nilable(String)).returns(T.nilable(Symbol)) }
+    def process_type(value = nil)
+      case T.unsafe(value)
       when nil
         @process_type
-      when PROCESS_TYPE_BACKGROUND, PROCESS_TYPE_STANDARD, PROCESS_TYPE_INTERACTIVE, PROCESS_TYPE_ADAPTIVE
-        @process_type = type.to_s
-      when String
+      when :background, :standard, :interactive, :adaptive,
+           "background", "standard", "interactive", "adaptive"
```

Comment:
I'd suggest we convert these to a consistent format so you don't need to duplicate.